howdoi/howdoi.py

```diff
@@ -332,11 +335,7 @@ def _get_questions(links):
     return [link for link in links if _is_question(link)]
 
 
-def _get_answer(args, links):
-    link = get_link_at_pos(links, args['pos'])
-    if not link:
-        return False
-
+def _get_answer(link, args):
```

Comment:
A lot of the `howdoi` functions (`_parse_cmd`, `_format_output`, etc) take `args` as the first argument so probably best to keep the same here.

Reply:
Right, will push a commit for these changes in a bit